src/adapters/default/attendance.adapter.ts

```diff
@@ -294,8 +295,78 @@ export class AttendanceService {
       data: result,
     });
   }
-  @Cron("0 10 15 * * 1-5")
-  async sendSMSNotificaiotns() {
-    console.log("sending SMS notification");
+  @Cron("0 00 18 * * 1-5")
+  public async sendSMSNotificaiotns() {
```

Comment:
Should this be in the adapter ? That will mean that each adapter will need to be aware of UCI implementation. Is that necessary ?